modules/core/src/main/java/io/fluo/recipes/map/Combiner.java

```diff
@@ -19,6 +19,5 @@
 import com.google.common.base.Optional;
 
 public interface Combiner<K, V, U> {
-  // TODO want to support deleting data
-  V combine(K key, Optional<V> currentValue, Iterator<U> updates);
+  Optional<V> combine(K key, Optional<V> currentValue, Iterator<U> updates);
```

Comment:
None of the CFM is documented yet, I was planning on doing that all at once.  I just created #26